puma_motor_driver/include/puma_motor_driver/driver.h

```diff
@@ -155,12 +155,13 @@ class Driver
 
     float interpretFixed8x8()
     {
-      return static_cast<int8_t>(data[1]) + static_cast<float>(data[0]) / 256.0f;
+      return static_cast<int8_t>(data[1]) + static_cast<float>(data[0]) / float(1<<8);
     }
 
     double interpretFixed16x16()
     {
-      return ((data[0] | static_cast<int32_t>(data[1]) << 8 | static_cast<int32_t>(data[2]) << 16 | static_cast<int32_t>(data[3]) << 24)) / double(1<<16);
+      return ((data[0] | static_cast<int32_t>(data[1]) << 8 |
+        static_cast<int32_t>(data[2]) << 16 | static_cast<int32_t>(data[3]) << 24)) / double(1<<16);
     }
   };
```

Comment:
I know you're pretty confident in them at this point, but it's never too late to start on a good habit— let's add a gtest covering these functions.